### PR TITLE
Fix for missing debuginfo package in debian/ubuntu and fixes in ci --build-pkgs

### PR DESCRIPTION
**Pull request by hirenvadalia**
#### Describe Bug or Feature

* When building PBS using `rpmbuild -ba` (+alien on Debian/ubuntu) it should generate debuginfo rpm/deb also, but as of now those packages are missing on Debian/Ubuntu due to missing `%debug_package` for Debian/Ubuntu as it's only enabled for SuSE/SLES (On RHEL/CentOS by default it's enabled)
* ci --build-pkgs depends on pbspro word for few files + missing CFLAGS while doing rpmbuild

#### Describe Your Change

* Enable `%debug_package` for Debian/Ubuntu also
* Fix ci --build-pkgs

#### Link to Design Doc

None

#### Attach Test and Valgrind Logs/Output

[before.txt](https://github.com/PBSPro/pbspro/files/4649833/before.txt)
[after.txt](https://github.com/PBSPro/pbspro/files/4649832/after.txt)

### CODE REVIEW

File: ci/build-pbs-packages.sh [hunk 1]
```diff
@@ -5,42 +5,40 @@
...
 cd /tmp/pbs
 ./autogen.sh
-cflags="-g -O2 -Wall -Werror"
-if [ "x${ID}" == "xdebian" -o "x${ID}" == "xubuntu" ]; then
-	cflags="${cflags} -Wno-unused-result"
-fi
-./configure CFLAGS="${cflags}" --prefix=/opt/pbs --enable-ptl
+mkdir target
+cd target
+../configure --prefix=/opt/pbs --enable-ptl
```
Comment (kjakkali) [inline ci/build-pbs-packages.sh:17]: Don't we need <cflags="-g -O2 -Wall -Werror"> here.

Comment (hirenvadalia) [reply ci/build-pbs-packages.sh:17]: No, here we run configure only for "make dist", rpmbuild will run configure again and at that time we need CFLAGS.

**Comment by sandisamp** [comment]
LGTM

Attaching build logs:
[debian_logs.txt](https://github.com/PBSPro/pbspro/files/4650218/debian_logs.txt)
[centos_logs.txt](https://github.com/PBSPro/pbspro/files/4650219/centos_logs.txt)
[ubuntu_18_logs.txt](https://github.com/PBSPro/pbspro/files/4650220/ubuntu_18_logs.txt)

**Comment by hirenvadalia** [comment]
Thanks @sandisamp for test logs...

**Pull request merged**